src/Waypoint/WaypointReaderSeeYou.cpp

```diff
@@ -299,6 +303,17 @@ WaypointReaderSeeYou::ParseLine(const TCHAR* line, Waypoints &waypoints)
     new_waypoint.comment = params[iDescription];
   }
 
+  if (iUserData < n_params) {
+    new_waypoint.details = params[iUserData];
+  }
+
+  if (iPics < n_params) {
+    for (tstring_view i : TIterableSplitString(params[iPics], _T(';'))) {
+      tstring p = _T("pics/");
+      p += i; 
```

Comment:
This can force XCSoar to read arbitrary files from the filesystem, just by writing something like "../../../" etc. into a waypoint file. XCSoar should validate file names loaded from data files.

Reply:
i.c. your point. Will validate the path to prevent directory traversal.

Reply:
Ideally this should be in file_embed because, this code also gets used by waypoint_details.txt

Reply:
See https://github.com/XCSoar/XCSoar/pull/833/commits/7c3fc9e653b595af70a5cf63e6dfdd441c148573